### DISCUSSION TITLE
Deck.gl on Google Colab via ObservableHQ

### DISCUSSION THREAD

**Discussion by cornhundred** [visgl/deck.gl#5654] [Show and tell]
Hi, all 

I recently found out about a Python library built by ObservableHQ, [observable-jupyter](https://pypi.org/project/observable-jupyter/), that allows you to use visualizations built in ObservableHQ in Jupyter notebooks without the Jupyter Widgets requirements. This seems to be useful for cases like Google Colab that do not currently support custom Jupyter Widgets such as the PyDeck widget (@ajduberstein)

* [Google Colab Notebook](https://colab.research.google.com/drive/1lXDh_AFgx3m9Qp307nz--1UwoBDkfbtH?usp=sharing)
    * this Colab notebook generates data for a scatterplot in Python and visualizes it using the deck.gl code in the ObservableHQ notebook
* [ObservableHQ](https://observablehq.com/@cornhundred/deck-gl-colab-test)
  * this ObservableHQ notebook has the deck.gl code that generates the scatterplot - the data variable is overwritten when Colab uses the notebook

**Comment by ajduberstein** [comment]
Thank you for bringing this up! [Making the changes to support this for pydeck](https://github.com/visgl/deck.gl/pull/5655).

**Reply by cornhundred** [reply]
No problem @ajduberstein  :) I'm wondering if it would be possible to run a MVT Tileserver from Google Colab and visualize the data from deck.gl's MVTLayer on Observable (using the observable-jupyter library) - I saw this sort of [similar example](https://talk.observablehq.com/t/running-google-colab-from-observable-or-vice-versa/4868/3) where they were hosting data from Google Colab unsing flask and ngrok. 

Were you able to get the PyDeck widget working on Colab?
